resources/assets/sass/modules/home-styling.scss

```diff
@@ -57,7 +57,12 @@
   }
 
   .home-buttons {
-    min-width: 50%;
+    min-width: 100%;
+
+  }
+
+  .home-buttons .pr-button {
```

Comment:
Here's an example where you could change this to a class named .home-button__pr-button